include/dwa_planner/dwa_planner.h

```diff
@@ -334,9 +359,9 @@ class DWAPlanner
 
   /**
    * @brief Calculate the command velocity
-   * @return The command velocity
+   * @return The command velocity (ROS 2 msg type)
```

Comment:
```suggestion
   * @return The command velocity
```
このコメントはなくて大丈夫です(ほかも同様)